main.tf

```diff
@@ -33,9 +33,10 @@ provider "apko" {
 provider "apko" {
   alias = "alpine"
 
-  default_archs      = [] # this defaults to "all"
   extra_repositories = ["https://dl-cdn.alpinelinux.org/alpine/edge/main"]
-  extra_packages     = ["ca-certificates-bundle"]
+  # These packages match chainguard-images/static
+  extra_packages = ["alpine-baselayout-data", "alpine-release", "ca-certificates-bundle"]
```

Comment:
I think these make sense at the provider level, to mimic what we do with wolfi (`wolfi-baselayout` and `ca-certificates-bundle`)

Reply:
Me too, can you scrub them from the other alpine `.apko.yaml` files?